actions/list-nrpe-checks

```diff
@@ -1,18 +1,20 @@
 #!/bin/bash
-
+  
```

Comment:
nit: please remove the trailing whitespace.

Reply:
Done